File: FSD.java
public class FSD {
    public static void main(String[] args) {
        // Fixed stack-dynamic array
        int[] arr = new int[5]; // Size fixed at declaration
        
        // Initialize array
        for (int i = 0; i < arr.length; i++) {
            arr[i] = i * 10; // Assign values to the array
        }
        
        // Print array
        System.out.println("Fixed Stack-Dynamic Array:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
    
}

File: Fixed Stack Dynamic/FSD.py
#Python lists are dynamic by nature, but we can simulate fixed behavior

def fixed_stack_dynamic():
    # Simulating fixed stack-dynamic array
    arr = [0] * 5  # Fixed size allocation
    
    for i in range(len(arr)):
        arr[i] = i * 10  # Assigning values to the array
    
    print("Fixed Stack-Dynamic Array (simulated):")
    print(arr)

fixed_stack_dynamic()